src/aioswitcher/device/__init__.py

```diff
@@ -376,3 +418,15 @@ def __post_init__(self) -> None:
         if self.device_type.category != DeviceCategory.SHUTTER:
             raise ValueError("only shutters are allowed")
         return super().__post_init__()
+
+
+@final
+@dataclass
+class SwitcherSingleShutterDualLight(SwitcherSingleShutterDualLightBase, SwitcherBase):
+    """Implementation of the Switcher Shutter with dual light device."""
+
+    def __post_init__(self) -> None:
+        """Post initialization validate device type category as SINGLE_SHUTTER_DUAL_LIGHT."""  # noqa: E501
```

Comment:
Please break into two lines instead of adding #noqa

Reply:
I tried to change to
"""Post initialization validate device type category
        as SINGLE_SHUTTER_DUAL_LIGHT.
        """

but got this lint errors:
src/aioswitcher\device\__init__.py:429:1: D205 1 blank line required between summary line and description
src/aioswitcher\device\__init__.py:429:1: D400 First line should end with a period

Reply:
That is correct, when documenting a function you should have a short description on the first line, than a black line and after you can multiple lines, something like this should work:
```
"""Post initialization

Validate device type category as SINGLE_SHUTTER_DUAL_LIGHT."""
```

Reply:
src/aioswitcher\device\__init__.py:429:1: D209 Multi-line docstring closing quotes should be on a separate line
src/aioswitcher\device\__init__.py:429:1: D400 First line should end with a period

Still issue..

Reply:
Well I just gave an example, yes first line should end with a period and you need to move the 3 """ to a new line.
It seems strange you come back here with this, wouldn't it be simple just to fix it the errors? they are clear.
To be honest I never face these errors since I am using linters which reformat the text as I write it.

Reply:
Done..